pipelines/common/src/main/java/com/google/fhir/analytics/view/ViewManager.java

```diff
@@ -63,9 +70,50 @@ public static ViewManager createForDir(String viewDefinitionsDir)
       ViewDefinition vDef = ViewDefinition.createFromFile(p);
       viewManager.viewMap.put(vDef.getResource(), vDef);
     }
+
+    Collection<ViewDefinition> viewDefinitions = viewManager.viewMap.values();
+    List<String> viewNames = new ArrayList<>();
+    for (ViewDefinition vDef : viewDefinitions) {
```

Comment:
I think this code can be simplified quite a bit; here are some suggestions:
- Change `viewNameMap` from a `Multimap` to a normal map (or please let me know if the `Multimap` functionality is needed beyond this method).
- Drop `viewNames` list; instead just before adding each name to `viewNameMap` check if that name already exists in `viewNameMap` and in that case add that name to `dupVews`.
- Drop `frequencyMap` and the two `for` loops below.

Reply:
Done.